components/atom/actionButton/src/index.scss

```diff
@@ -0,0 +1,210 @@
+@import '~@s-ui/theme/lib/index';
+@import '~@s-ui/react-atom-icon/lib/index';
+
+$base-class: '.react-AtomActionButton';
+
+$bdsz-atom-action-button-icon-outline: 1px;
+
+$c-atom-action-button-neutral: color-variation($c-gray, 2);
+
+$c-accent-active: color-variation($c-accent, -2) !default;
```

Comment:
We should use a better naming for variables, as this could easily collide.

I think this should be something like this:
````
$c-atom-action-button-accent-active```

Reply:
thanks! done!